src/View/Admin/index.html.twig

```diff
@@ -2,4 +2,21 @@
 
 {% block content %}
     <h2>Connexion</h2>
+
+  <div class="container login">
+        <form method="POST" action="">
```

Comment:
sorry you will curse me but we have said that we do not deal with authentication (no login/password form) 
please remove it (j'écris en anglais pour que ça passe mieux :p )

Reply:
Removed!  :-)

on va arriver, comme tu dis.